### ISSUE TITLE
Can i ask something?

### ISSUE THREAD

**Issue by kevin0598** [adafruit/RadioHead#48]
Is this compatible library to use On Arduino Nano V3 with ATmega328??

**Comment by EmilM256** [comment]
Yes. I'm using Nano V3 5V with a logic level converter in my project. The radio module is RFM95. Works well.
